universal-login-react/src/ui/commons/GasPrice/GasPrice.tsx

```diff
@@ -115,9 +116,18 @@ export const GasPrice = ({isDeployed = true, deployedWallet, sdk, gasLimit, onGa
       </div>
     </>
   );
+
+  const returnRenderComponentOrSpinner = (gasModes?: GasMode[]) => {
+    return (gasModes ? renderComponent(gasModes) : <Spinner className="spinner-small" />);
+  };
+
+  if (sdk?.isRefundPaid()) {
+    onGasParametersChanged({gasToken: 'ether', gasPrice: bigNumberify('0')});
```

Comment:
Btw. ether in ticket description was a mental shortcut. GasToken, in general, is the address of the token. So in ether case, its value is `0x000..0`.